samples/mixed/consumer/src/main.rs

```diff
@@ -55,9 +61,9 @@ fn start_show_notification_repeater(provider_uri: String, consumer_uri: String)
                 Err(status) => warn!("{status:?}"),
             }
 
-            info!("Invoked the show-notification command on endpoint {provider_uri}");
+            debug!("Completed the invoke request");
 
-            sleep(Duration::from_secs(5)).await;
+            sleep(Duration::from_secs(55)).await;
```

Comment:
should these sleep durations be configurable?

Reply:
No